static/app/utils/replays/hooks/useMouseTracking.tsx

```diff
@@ -77,19 +77,25 @@ function MouseTracking({onMouseMove, children}: Props) {
       controller.current = new AbortController();
     }
 
-    onMouseMove(undefined);
-  }, [onMouseMove, controller]);
+    onPositionChange(undefined);
+  }, [onPositionChange, controller]);
 
-  return (
-    <div
-      ref={elem}
-      onMouseEnter={handleMouseMove}
-      onMouseMove={handleMouseMove}
-      onMouseLeave={onMouseLeave}
-    >
-      {children}
-    </div>
-  );
+  return {
+    ref: elem,
+    ...rest,
```

Comment:
Should we destructure the `onMouse*` handlers? This way it would remove the possibility of someone moving the `...rest` below the properties that you're trying to override.